portfolio.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="portfolio.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.1/css/all.css" />
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="topnav">
+            <div>
+                <a class="mylogo" href="#mylogo">My Logo</a>
+            </div>
+            <ul class="nav__links">
+                <!-- <li class="line"><a href="#portfolio">Portfolio</a></li>
+                <li class="line"><a href="#about">About</a></li>
+                <li class="line"><a href="#contact">Contact</a></li> -->
+                <li class="line"><a href="#portfolio"></a></li>
+                <li class="line"><a href="#about"></a></li>
+                <li class="line"><a href="#contact"></a></li>
+
+            </ul>
+        </nav>
+    </header>
+    <div class="backimage">
+        <div class="welcome">
+            <p>I am Amrendra Glad to see you!</p>
```

Comment:
- [ ] Did you know that you can find all the style details in figma if you click on elements? If no then lets try to do it! click on element and check the right section. You will see all the specifications like font size, colors etc. Pleaase try to check the styles for heading and paragraph of the page and apply them on your project!

First Header  | Your heading and paragraph 
------------- | -------------
<img width="378" alt="Screenshot 2022-02-10 at 21 14 05" src="https://user-images.githubusercontent.com/77038610/153460439-72f120ac-6d17-43bf-8a3c-3597d4ca17f6.png"> | <img width="380" alt="Screenshot 2022-02-10 at 21 13 22" src="https://user-images.githubusercontent.com/77038610/153460302-5a2ce9ed-4366-4725-b723-4256aa6ec163.png">
Please set the same colors and font styles 😉 

- [ ] Since I am `Amrendra Glad to see you!` is a heading, and every page should have an `<h1>` tag, please wrap the heading with `<h1>` tag.